packages/bpk-tokens/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bpk-tokens",
-  "version": "27.1.0",
+  "version": "27.1.0-beta.5",
```

Comment:
Shouldn't we let Lerna set this? It caused an issue when we updated this manually yesterday.